assets/language/pt-br.json

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
since pt.json aleady exists, please delete this file

Reply:
done